Samples/Scratchpad/scratchpad_c_single_file/main.c

```diff
@@ -4,20 +4,19 @@
 #include "header_test2.h"
 #include "header_test1.h"
 
-typedef struct {
+typedef struct ScratchpadCSharp_SingleFileTests_SomeStruct {
 	int32_t Value;
 } ScratchpadCSharp_SingleFileTests_SomeStruct;
 
 int32_t ScratchpadCSharp_SingleFileTests_NumberArray[10] = {1,2,3,4,5,6,7,8,9,10};
 
-#define addOneMacro(a) ((a) + 1)
 int32_t main(void) {
 	ScratchpadCSharp_SingleFileTests_SomeStruct __local_0 = {0};
 	(*(&__local_0)) = ((ScratchpadCSharp_SingleFileTests_SomeStruct){0});
 	((&__local_0)->Value) = 33;
 	do_stuff2();
 	do_stuff();
-	addOneMacro(10);
+	addOneMacro((int32_t*)10);
```

Comment:
Need to figure out why this is is casting it to a pointer and the macro is being removed.  This is probably a bug

Reply:
fixed this